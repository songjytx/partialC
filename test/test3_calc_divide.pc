void main(){
	int a = 4 / 2;
	printi(a);
	float b = 6.2 / 3.0;
	printf(b);
}