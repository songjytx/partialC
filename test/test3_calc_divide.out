2
2.066667
